### DISCUSSION TITLE
Jsonrprsee Notification Spec

### DISCUSSION THREAD

**Discussion by aniketfuryrocks** [paritytech/jsonrpsee#1108] [JSONRPC migration guide]
Hi, I was working on porting the pubsub tests of solana from jsonrpc to jsonrpsee. I noticed that the tests expect the `"jsonrpc": 2.0` field to be there while the new jsonrpsee crate doesn't return it in the response. Is this valid jsonrpc ?

*This works*
![image](https://user-images.githubusercontent.com/42654595/236649800-c710d7b6-3fda-417a-9ca5-8812cc76c977.png)

**Comment by niklasad1** [comment]
https://github.com/paritytech/jsonrpsee/issues/1045,

The "jsonrpc" field can be omitted but it's an `Option` so should be possible enable.
Perhaps it was changed unintentionally would be better to send out notifications with the `jsonrpc` field by default.

Can you open an issue and write how to reproduce it?

**Reply by aniketfuryrocks** [reply]
https://github.com/paritytech/jsonrpsee/issues/1109
I'll add code to reproduce
